Jellyfin.Api/Controllers/UniversalAudioController.cs

```diff
@@ -201,7 +204,7 @@ public class UniversalAudioController : BaseJellyfinApiController
                     Static = isStatic,
                     PlaySessionId = info.PlaySessionId,
                     // fallback to mpegts if device reports some weird value unsupported by hls
-                    SegmentContainer = Array.Exists(supportedHlsContainers, element => element == transcodingContainer) ? transcodingContainer : "mpegts",
+                    SegmentContainer = Array.Exists(supportedHlsContainers, element => element == transcodingContainer) ? transcodingContainer : "ts",
```

Comment:
Does this change affect the API?

Reply:
nope. It is just a file extension per Apple's docs.
Let `ts` as default container since hls muxer only support mpegts and fmp4.